src/pages/Lending/hooks/useLendingPositionData.tsx

```diff
@@ -19,7 +19,8 @@ export const useLendingPositionData = ({ accountId, assetId }: UseLendingPositio
   const poolAssetMarketData = useAppSelector(state => selectMarketDataById(state, assetId))
 
   const lendingPositionData = useQuery({
-    staleTime: Infinity,
+    // 2 minutes before the data is considered stale, meaning firing this query will trigger queryFn
+    staleTime: 120,
```

Comment:
likes like staleTime is millis, not seconds
https://tanstack.com/query/v4/docs/react/guides/initial-query-data#staletime-and-initialdataupdatedat

Reply:
🧠 4d9904c664